src/ophys_etl/modules/suite2p_registration/suite2p_utils.py

```diff
@@ -241,6 +243,8 @@ def optimize_motion_parameters(initial_frames: np.ndarray,
         Number of frames to disregard from the start of the movie. Default 0.
     trim_frames_start : int, optional
         Number of frames to disregard from the end of the movie. Default 0.
+    Number of batches to load. Total returned size is
```

Comment:
Forgot to include the parameter name here

Reply:
Also: since I had to pause and ask "why are we doing that?", can you add a comment somewhere explaining that we are loading the frames in batches (rather than loading, say 10,000 evenly space frames) so that we get a representation of the motion from one frame to the next? Just want to prevent future us from doing something foolish.

Reply:
I don't actually see the explanatory comment about why we are loading n_batches instead of n evenly spaced frames.

I am correct in my interpretation, true? It's so that we can preserve the small timescale motions in the original movie?

Reply:
It should be this line right or is there additional comments you are looking for?

"Number of batches to load. Processing a large number of frames at once will likely result in running out of memory, hence processing in batches. Total returned size isn_batches * suit2p_args['batch_size']."

Reply:
Ah. Okay. I misinterpreted the reason for processing in batches. I thought it was something more complicated than it was.

LGTM